src/account/views.py

```diff
@@ -113,6 +114,17 @@ def register(request):
             # doesn't exist. If the database cannot be created, that handler
             # will throw an exception.
             user = datahub_authenticate(username, password)
+
+            # Create row level security policies in the dh_public policy table
+            # granting user select, insert, update access to policies he create
+            rls_manager = RowLevelSecurityManager(
+                'dh_public', 'policy', 'dh_public', 'dh_public')
```

Comment:
These should be done via static methods, and the usernames shouldn't be hardcoded. (This is pretty much just a note for myself.)